telegram/_message.py

```diff
@@ -663,6 +663,10 @@ class Message(TelegramObject):
     .. |custom_emoji_no_md1_support| replace:: Since custom emoji entities are not supported by
        :attr:`~telegram.constants.ParseMode.MARKDOWN`, this method now raises a
        :exc:`ValueError` when encountering a custom emoji.
+
+    .. |blockquote_no_md1_support| replace:: Since block quotation entities are not supported
+       by :attr:`~telegram.constants.ParseMode.MARKDOWN`, this method now raises a
+       :exc:`ValueError` when encountering a block quotation.
```

Comment:
This sounds sane to me, since there is no kind of fallback option. Just for completeness I want to mention that for the custom emojis (where we did have a fallback option), we did not raise an exception immediately, see https://github.com/python-telegram-bot/python-telegram-bot/pull/3673/files#diff-a9dec124bab0bbe6efcdb93c525ae03d28c86809642c8550d2057b19db8d7bf3. cc-ing @harshil21 @clot so that other dev team members can give their opinion as well

Reply:
Agree with that, it is called as version "1" for a reason, raising the error is the way to go specially after we did that warning also

Reply:
Agree!